### PR TITLE
Update feature-flags.md

### PR DESCRIPTION
**Pull request by joshjung**
Some minor updates to punctuation, code examples, and language to the feature flags Markdown file

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/plugins/feature-flags.md [hunk 1]
```diff
@@ -6,39 +6,40 @@ description: Details the process of defining setting and reading a feature flag.
...
+This page describes the process of defining, setting and reading a feature flag. If you are looking for using feature flags with software templates that can be found under [Writing Templates](https://backstage.io/docs/features/software-templates/writing-templates#remove-sections-or-fields-based-on-feature-flags).
 
 ## Defining a Feature Flag
 
 ### In a plugin
 
-Defining feature flag in a plugin is done by passing the name of the feature flag into the `featureFlags` array:
+Defining a feature flag in a plugin is done by passing the name of the feature flag into the `featureFlags` array:
 
 ```ts
```
Comment (awanlin) [inline docs/plugins/feature-flags.md:17]:
```suggestion
```ts title="src/plugin.ts"
```

Comment (awanlin) [reply docs/plugins/feature-flags.md:17]: If we do this then we can remove the comment at line 18 just below

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this @joshjung, just one small nit I didn't notice in the other PR

**Review by awanlin — APPROVED** [review]
Awesome! 🚀

**Review by vinzscam — APPROVED** [review]

**Review by vinzscam — APPROVED** [review]
thanks!

**Pull request merged**